ci/tests/vmachine/vmachine_test.py

```diff
@@ -123,96 +123,75 @@ def check_scrubbing_test():
-            GeneralVMachine.logger.info("Starting RAW disk creation")
-            out, err, _ = General.execute_command('qemu-img convert -O raw {0}{1} /mnt/{2}/{3}.raw'.format(template_folder, image_name, vpool_name, disk_name))
-            if err:
-                GeneralVMachine.logger.error("Error while creating raw disk: {0}".format(err))
-
-        for vm_number in range(nr_of_disks):
-            machine_name = "machine-{0}".format(vm_number)
-            disk_name = "scrubdisk-{0}".format(vm_number)
-            GeneralVMachine.logger.info("Starting vmachine creation from RAW disk")
-            out, err, _ = General.execute_command('virt-install --connect qemu:///system -n {0} -r 512 --disk /mnt/{1}/{2}.raw,'
-                                                  'device=disk --noautoconsole --graphics vnc,listen=0.0.0.0 --vcpus=1 --network network=default,mac=RANDOM,'
-                                                  'model=e1000 --import'.format(machine_name, vpool_name, disk_name))
-            if err:
-                GeneralVMachine.logger.error("Error while creating vmachine: {0}".format(err))
-
-        def snapshot_vdisks():
-            vds = GeneralVDisk.get_vdisks()
-            for disk in vds:
-                metadata = {'label': 'snap-' + disk.name,
-                            'is_consistent': True,
-                            'timestamp': time.time(),
-                            'machineguid': disk.vmachine_guid,
-                            'is_automatic': False,
-                            'is_sticky': False}
-                VDiskController.create_snapshot(disk.guid, metadata)
-
+        disk_name = "scrubdisk"
+        GeneralVMachine.logger.info("Starting RAW disk creation")
+        out, err, _ = General.execute_command('qemu-img convert -O raw {0}{1} /mnt/{2}/{3}.raw'.format(template_folder, image_name, vpool_name, disk_name))
+        if err:
+            GeneralVMachine.logger.error("Error while creating raw disk: {0}".format(err))
+
+        def snapshot_vdisk(vdisk):
+            metadata = {'label': 'snap-' + vdisk.name,
+                        'is_consistent': True,
+                        'timestamp': time.time(),
+                        'machineguid': vdisk.vmachine_guid,
+                        'is_automatic': False,
+                        'is_sticky': False}
+            VDiskController.create_snapshot(vdisk.guid, metadata)
+
+        counter = 60
+        while counter and vdisk is None:
+            time.sleep(5)
+            vdisks = GeneralVDisk.get_vdisk_by_name(disk_name)
+            if len(vdisks):
+                vdisk = vdisks[0]
+            counter -= 5
+        assert counter > 0, "Vdisk with name {0} didn't appear in the model after 60 seconds".format(disk_name)
         # snapshoting disks for the first time
-        snapshot_vdisks()
-        counter = timeout / timer_step
+        snapshot_vdisk(vdisk)
+        counter = 100
         while counter > 0:
```

Comment:
use counter / step here too
even better, put constants for this at the beginning of the method and use those later on